lib/vanagon/component/dsl.rb

```diff
@@ -90,7 +91,7 @@ def environment(&block)
       # @param strip [String, Integer] directory levels to skip in applying patch
       # @param fuzz [String, Integer] levels of context miss to ignore in applying patch
       def apply_patch(patch, strip: 1, fuzz: 0)
-        @component.patches << OpenStruct.new('path' => patch, 'strip' => strip.to_s, 'fuzz' => fuzz.to_s)
+        @component.patches << Vanagon::Patch.new(patch, strip, fuzz)
```

Comment:
@mckern actually, that PR was merged into the vanagon_for_the_win branch - do we want to just cherry pick that commit onto master or something?